docs/introduction.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Introduction
+id: introduction
+---
+
+# What is `granary`?
+
+Granary is a job runner for cloud-based geospatial machine learning.
+Its goal is to simplify running models and to track and serve
+the results of predictions. It puts a REST API between you and AWS Batch
+to simplify interactions that otherwise involve repeatedly checking AWS SDK
+documentation.
+
+## Background info
+
+To begin with, here are some IDs and imports that we'll be using throughout the examples:
+
+```scala mdoc
+import com.rasterfoundry.granary.datamodel._
+import io.circe.syntax._
+import java.time.Instant
+import java.util.UUID
+
+val modelId = UUID.fromString("@MODEL_ID@")
+val predictionId = UUID.fromString("@PREDICTION_ID@")
+val webhookId = UUID.fromString("@WEBHOOK_ID@")
+val invocationTime = Instant.ofEpochMilli("@INVOCATION_TIME@".toLong)
+```
+
+The `modelId` is a faked ID that you can imagine represents a stored `Model`
+in the database. The same is true for the `predictionId`, but for a `Prediction`.
+The `webhookId` represents a single-use webhook that can be used for status updates
+on a prediction. The `invocationTime` is a pretend time that this prediction was
+kicked off. The imports are collectively present to make the rest of the
+documentation compile. You can ignore them.
```

Comment:
Consider leading the section with this last sentence so that the reader is told that they can ignore the section before they have read to the bottom.